Gameday URL Aliases

phil-lopreiato opened the pull request:
Allows us to have urls like /gameday/fun that automatically embed the FUN stream. The pull:
 - Tweaks the structure of the GD sitevar to store special webcasts and aliases
 - Adds an admin panel page to create/delete aliases & special webcasts
 - Make a request handler for /gameday/blah and /watch/blah to do URL transforms

![screenshot from 2016-12-03 20-25-57](https://cloud.githubusercontent.com/assets/2754863/20863396/f19e3b06-b996-11e6-9ccf-0fe2c81cb6ff.png)

fangeugene commented on templates/admin/gameday_dashboard.html:
```diff
@@ -0,0 +1,80 @@
+<!-- Row to add another special webcast -->
+<tr>
+    <td><input type="text" class="form-control" name="webcast_name" form="add_webcast" placeholder="Display Name"/></td>
+    <td><input type="text" class="form-control" name="webcast_type" form="add_webcast" placeholder="Stream Type"/></td>
```
This should be a dropdown. Look at the existing stuff.

phil-lopreiato commented:
Updated with a few things:
 - Use dropdown for webcast types
 - Added event and team watch aliases (/watch/2016cmp and /watch/frc1124). Goes to blank gameday if something isn't valid
 - Put gameday controllers in their own file

![screenshot from 2016-12-03 21-26-53](https://cloud.githubusercontent.com/assets/2754863/20863643/57b2c2d8-b99f-11e6-94ff-37873731004d.png)
